cineast-api/src/main/java/org/vitrivr/cineast/api/rest/handlers/abstracts/ParsingActionHandler.java

```diff
@@ -40,24 +44,32 @@
      * @throws Exception implementation can choose to throw exception
      */
     @Override
-    public Object handle(Request request, Response response) throws Exception {
-        Map<String, String> params = request.params();
+    public void handle(Context ctx) throws Exception {
+    	Map<String, String> params = ctx.pathParamMap();
         if (params == null) {
             params = new HashMap<>();
         }
-        response.type("application/json");
-        switch (request.requestMethod()) {
+        ctx.contentType("application/json");
+        switch (ctx.method()) {
             case "GET":
-                return MAPPER.writeValueAsString(this.doGet(params));
+                ctx.result(MAPPER.writeValueAsString(this.doGet(params)));
```

Comment:
Why is JSON conversion done explicitly, rather than just using the functionality Javalin provides internally?